src/components/VenueGuideMap/VenueGuideMapDialog/index.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogClose,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog";
+
+export const VenueGuideMapDialog = () => {
+  return (
+    <Dialog>
+      <DialogTrigger asChild>
+        <Button
+          variant="default"
+          size="lg"
+          className="rounded-full h-[60px] bg-blue-purple-500 hover:bg-blue-purple-600 text-white px-8"
+        >
+          <span className="text-18 md:text-22 leading-[1.8]">
+            マップを拡大する
+          </span>
+        </Button>
+      </DialogTrigger>
+      <DialogContent className="w-11/12 max-w-7xl bg-white flex flex-col items-center p-1 pb-4">
+        <DialogHeader className="sr-only">
+          <DialogTitle>会場マップ</DialogTitle>
+          <DialogDescription>会場マップの拡大図です。</DialogDescription>
+        </DialogHeader>
+        <div className="hidden md:block">
+          <img src="/venue-guide-map-h.png" alt="PC用会場図" />
```

Comment:
細かくて申し訳ないですが、会場マップを補足するようなテキスト情報は他にないので代替テキストを充実させたいです🙏
（他の箇所も同様）
AIがいい感じに会場を説明する代替テキストを生成してくれたので参考にしてもらえればと思います！

```suggestion
          <img src="/venue-guide-map-h.png" alt="イベント会場の2階と3階のマップ。2階にはトグルルーム、配布物コーナー、受付・本部、スポンサー ブース、スタンプラリーゴールがあり、一方通行の通路がある。3階にはアセンドトラック、レバレジーズトラック、スポンサー ブース、Ask the Speaker、3階案内、休憩スペース＆企画エリアがある。一部通行不可エリアも示されている。各階にエレベーターとトイレがある。" />
```